setuptools_rust/build.py

```diff
@@ -282,7 +282,7 @@ def build_extension(self, ext):
                     except subprocess.CalledProcessError as e:
                         pass
 
-            if executable:
+            if executable or sys.platform == "win32" or sys.platform == "cygwin":
```

Comment:
You mean we need `chmod` for dlls?

Reply:
Yes. For example, [tokenizers.dll](https://pypi.org/project/tokenizers/) needs to `chmod u+x` when installing on Cygwin.

Reply:
- How about doing this hack only for cygwin? Is it also required for win32 platform?
- Please add a comment that explains why cygwin needs it.

Reply:
This is for "cross-compile" between win32 and cygwin. Now I'm trying to compile PyO3 dll's for win32 with cygwin gcc, and then make wheels for win32 with setuptools_rust. In such cases, I need `sys.platform == "win32"` here. I agree that it is very rare case now.

Reply:
I think that MSVC always outputs dll's with `-rwxr-x---` modes, so they don't need `chmod` here explicitly.